### PR TITLE
Node manager

### PR DESCRIPTION
**Pull request by MitaliBo**

### CODE REVIEW

File: pyrsia_node/src/node_manager/handlers.rs [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        0xec, 0xa7,
+    ];
+
+    #[test]
+    fn put_and_get_artifact_test() -> Result<(), anyhow::Error> {
+        println!("put_and_get_artifact_test started !!");
+
+        // tetst artifact file in resources/test dir
+        let mut curr_dir = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+        curr_dir.push("resources/test/artifact_test.txt");
```
[the remark this answers is not in the document]
Comment (prince-chrismc) [reply pyrsia_node/src/node_manager/handlers.rs:72]: I like how it's done in https://github.com/pyrsia/pyrsia/pull/82

[the remark this answers is not in the document]
Comment (MitaliBo) [reply pyrsia_node/src/node_manager/handlers.rs:72]: yeah that is good idea but i think i am done taking suggestions for this PR as its getting messy, probably will create another one...these suggestions are not a blocker

**Comment by prince-chrismc** [comment]
@MitaliBo please fix the linting issues

The `use` is in the wrong place since it does not like the comments at the top of the file

**Comment by MitaliBo** [comment]
Replaces #90

**Pull request closed without merging**